Sources/Logging.swift

```diff
@@ -17,7 +17,7 @@ import Darwin.C.stdio
 private let loggingTag = "Swift"
 
 @discardableResult
-@_silgen_name("__android_log_write")
+@_cdecl("__android_log_write")
 public func android_log_write(_ prio: Int32, _ tag: UnsafePointer<CChar>, _ text: UnsafePointer<CChar>) -> Int32
```

Comment:
are you sure you want to use _cdecl here? i think _cdecl cannot annotate empty functions
https://github.com/apple/swift/blob/master/test/attr/attr_cdecl.swift#L9